tools/ci/docker/Dockerfile

```diff
@@ -2,7 +2,11 @@ FROM ghcr.io/graalvm/jdk-community:21
 
 USER root
 
-ENV LOG_LEVEL=INFO
+ARG LOG_LEVEL=info
+ARG RPC_PORT=30001
+ARG DATA_PORT=30002
+ARG PRINT_VERSION=0
+ARG JAVA_OPTS="-XX:MaxRAMPercentage=90.0 -XX:InitialRAMPercentage=90.0"
```

Comment:
How did you find these are the right options? Do you have any reference?

Reply:
Past experience but not tested in battlefield with Enso. Gives enough room when JVM decides to go over the limit (which it occasionally can do).